cms/djangoapps/contentstore/course_group_config.py

```diff
@@ -363,3 +363,21 @@ def get_or_create_content_group(store, course):
             content_group_configuration
         )
         return content_group_configuration
+
+    @staticmethod
+    def get_all_user_partitions(store, course):
+        """
+        Returns all the available content groups with updated group usage information
```

Comment:
Update doc.

And the name of the method is confusing with "get_all_partitions_for_course", etc. What is special about this method? Is it that it returns JSON and has the usage information?

How about get_all_user_partition_details or get_all_partitions_with_usage?